lib/jxl/splines.cc

```diff
@@ -589,6 +590,10 @@ Status Splines::InitializeDrawCache(const size_t image_xsize,
       return JXL_FAILURE(
           "identical successive control points in spline %" PRIuS, i);
     }
+    splines.push_back(spline);
+  }
+  for (size_t i = 0; i < splines_.size(); ++i) {
```

Comment:
Could also write it like this:

```suggestion
  for (Spline& spline: splines) {
```

Reply:
sure!